content/pt-br/docs/tasks/configure-pod-container/assign-cpu-resource.md

```diff
@@ -0,0 +1,275 @@
+cpu-demo                    974m         <something>
+```
+
+Lembre-se que ao especificar `-cpu "2"`, você configurou o Contêiner para tentar 
+usar 2 CPUs, mas o Contêiner está apenas habilitado a usar cerca de 1 CPU. 
+O uso de CPU dos contêineres está sendo estrangulado, 
+porque o contêiner está tentando usar mais recursos de CPU que o seu limite. 
+
+{{< note >}}
+Outra possível explicação para o uso de CPU estar abaixo de 1,0, é que o Nó 
+não dispõe de recuros de CPU suficientes disponíveis. Lembre-se que o pré-requisito 
+para este exercício, é que o seu cluster tenha disponível ao menos 1 CPU para uso. 
+Se o seu Contêiner executa em um Nó que tem apenas 1 CPU, o Contêiner não pode usar 
+mais que 1 CPU, sem contar o limite de CPU especificado para o Contêiner. 
+
+{{< /note >}}
+
+## Unidades de CPU
+
+Os recursos de CPU são medidos em unidades de *CPU*. Uma CPU, em Kunernetes, é equivalente a:
+
+* 1 AWS vCPU
+* 1 GCP Core
+* 1 Azure vCore
+* 1 Hyperthread em um processador Intel bare-metal com Hyperthreading
+
+
+Valores fracionários são aceitos. Ao Contêiner que requer 0,5 CPU é garantido 
+pelo menos a metade da CPU que um Contêiner que requer 1 CPU. 
+Você pode usar o sufixo m para representar milli. Por exemplo 100m CPU, 100 milliCPU, 
+e 0,1 CPU são todas a mesma. Precisão menor que 1m não é aceita.
+
+CPU é sempre requisitada como uma quantidade absoluta, nunca como uma quantidade relativa; 0,1 é a mesma quantidadede de CPU em uma máquina single-core, dual-core, ou 48-core.
+
+Apague seu Pod:
+
+```shell
+kubectl delete pod cpu-demo --namespace=cpu-example
+```
+
+## Especifique um requisito de CPU que é muito grande para seus Nós
+
+Requisitos e limites de CPU são associados aos Contêineres, mas é funcional pensar num Pod como tendo requisito e limite de CPU. O requisito de CPU para um Pod é a soma dos requisitos de CPU de todos os Contêineres no Pod. Da mesma forma, o limite de CPU para um Pod é a soma dos limites de CPU para todos os Contêineres no Pod. 
+
+O agendamento do Pod é baseado em requisições. Um Pod é agendado para executar em um Nó, somente se o Nó possuir suficiente recursos de CPU disponíveis para satisfazer os requisitos de CPU do Pod.
+
+Neste exercício, você cria um Pod que tem um requisito de CPU tão grande, que excede a capacidade de qualquer Nó em seu cluster. Aqui está o arquivo de configuração para o Pod que tem um Contêiner. O Contêiner requer 100 CPU, que provávelmente excede a capacidade de qualquer Nó em seu cluster.
+
+{{< codenew file="pods/resource/cpu-request-limit-2.yaml" >}}
+
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/resource/cpu-request-limit-2.yaml --namespace=cpu-example
+```
+
+Veja o status do Pod:
+
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+```
+
+A saída mostra que o status do Pod é Pending. Que significa, que o Pod não está 
+agendado para executar em qualquer Nó, e permanecerá no estado Pendente indefinidamente.
```

Comment:
These two sentences are lacking a bit of flow to me. It feels a word is missing before "Que significa" and there's an extra comma afterwards.